rebar.config.script

```diff
@@ -1,8 +1,10 @@
 case {os:getenv("GITHUB_ACTIONS"), os:getenv("GITHUB_TOKEN")} of
   {"true", Token} when is_list(Token) ->
     CONFIG1 = [{coveralls_repo_token, Token},
-               {coveralls_service_job_id, os:getenv("GITHUB_RUN_ID")},
+               {coveralls_service_number, os:getenv("GITHUB_RUN_ID")},
                {coveralls_commit_sha, os:getenv("GITHUB_SHA")},
+               {coveralls_parallel, true},
```

Comment:
I remember seeing this flag, but it's curious that we don't need to set it in emqx :thinking: